knowledge_graph_queries/openapi.yml

```diff
@@ -0,0 +1,133 @@
+openapi: "3.0.1"
```

Comment:
The `openapi.yml` file needs to be under the `openapi` folder because of `specification_dir='openapi/'`, please restore the folder and place this file there.

Reply:
oki